deploy/crds/crd-certificates.yaml

```diff
@@ -285,6 +285,7 @@ spec:
                     description: CertificateCondition contains condition information for an Certificate.
                     type: object
                     required:
+                      - observedGeneration
```

Comment:
We can't make this field required as older versions of cert-manager/older clients won't set it

Reply:
Yep, this is optional as per the standardised conditions KEP: https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/1623-standardize-conditions#proposal